aws/templates/aws-template.yaml

```diff
@@ -8,7 +8,7 @@ Parameters:
         Default: ""
     APIUrl:
         Type: "String"
-        Description: "Data Url of vRealize Log Insight Cloud"
+        Description: "Data Url of vRealize Log Insight Cloud or Wavefront proxy(Note: APIToken is not required for Wavefront proxy URL)"
```

Comment:
Need Amit K to review this description.

Reply:
Also this commit doesn't have changes around API Token validation. So isn't this description inaccurate?

Reply:
I will share the draft PR with Amit K.